### ISSUE TITLE
Anomalous (big) values on graphite/carbon 

### ISSUE THREAD

**Issue by piccardi** [netdata/netdata#5104]
##### Bug report summary

Sending data to a graphite/(carbon) backend I got anomalous (very big) values

##### OS / Environment

Debian Stretch on both monitored server and carbon/graphite/grafana server; netadata installed from source, updated with ./netdata-updater.sh, version v1.12.0-rc1-34-gd5fb2be8 (but I got the same with 1.11). graphite-carbon installed from official debian packages (0.9.15-1), grafana 5.4.2 from debian packages on grafana repository.

##### Component Name

backends

##### Steps To Reproduce

Not sure if it's a bug or a misconfiguration on my side,  I tryed to follow the docs but I'm not so deep in to carbon, grafana and netdata.

This my netdata.conf configuration, the only changes parts from installed default are the following:

```
[registry]
registry to announce = http://remote.host.istance:19999
[backend]
        enabled = yes
        type = graphite
        destination = tcp:remote.carbon.instance:2003
        data source = as collected
        update every = 60
```

this is the result in grafana using carbon as datasource for the cpu load1 and load5 average:

![grafanacpuload](https://user-images.githubusercontent.com/14810174/50601941-17582480-0eb6-11e9-94bd-21dc055736b2.png)

but also looking at the data I got:

```
~# whisper-info /var/lib/graphite/whisper/netdata/haynes/system/load/load1.wsp 
maxRetention: 86400
xFilesFactor: 0.5
aggregationMethod: average
fileSize: 17308

Archive 0
retention: 86400
secondsPerPoint: 60
points: 1440
size: 17280
offset: 28
``` 

and:

```
~# whisper-fetch /var/lib/graphite/whisper/netdata/haynes/system/load/load1.wsp
 ...
1546447500	880.000000
1546447560	800.000000
1546447620	580.000000
1546447680	1320.000000
1546447740	1250.000000
1546447800	1110.000000
1546447860	640.000000
1546447920	350.000000
1546447980	330.000000
1546448040	None

```
so getting thousand for  a server with normal load:

##### Expected behavior

I'd expected to have values similar to the one showed by netdata:

![netdata](https://user-images.githubusercontent.com/14810174/50602660-23dd7c80-0eb8-11e9-95ed-26aee318cb75.png)

Regards
Simone

**Comment by cakrit** [comment]
The following setting asks that the metrics are sent as they are (raw). 
```
data source = as collected
```

To get the `average` you see in the netdata chart, you will need to use 

```
data source = average
```

See point 4 [here](https://docs.netdata.cloud/backends/#features)

**Comment by piccardi** [comment]
Thanks for the suggestion, switching to average got me some reasonable values.

But I have to say that the description of the ```as collected``` value in the docs you are pointing to (I read those before) has mislead me, because reading that you are sending  1 sample over 10 make me to  understand it as a single value of the load average (what I see in ```/proc/loadavgs```) is sent (instead of the average). 

I cannot see what raw means in this context (probably bacause I never used collectd or telegraf and I'm missing some backgroud) and why it is different from the plain value, so I think a little more information about what is sent in the ```as collected``` case is needed.

**Comment by cakrit** [comment]
Thanks for the feedback, I'd like to improve the documentation.
It currently says the following:

```
as collected: the latest collected value is sent to the backend. This means that if netdata is configured to send data to the backend every 10 seconds, only 1 out of 10 values will appear at the backend server. The values are sent exactly as collected, before any multipliers or dividers applied and before any interpolation. This mode emulates other data collectors, such as collectd or telegraf.
```

How do you think it could be improved? e.g would this have helped you?

```
The values are sent exactly as collected, i.e. raw, unprocessed values, before any multipliers or dividers applied and without any interpolation/calculation of average values etc.
```

**Comment by piccardi** [comment]
Yes that way is better, I'd add also "so they not match to the ones shown on the netdata charts". And I will put the bold phrase in the ```average``` description at the beginning as a general advice, with something like:

4. Three modes of operation, for all backends (**if you don't know
which mode to use, use average**):
